docs/index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   hpe-csi-driver:
+  - apiVersion: v1
+    appVersion: 1.4.1
```

Comment:
@raunakkumar for future cosmetic updates to the chart that doesn't increment the HPE CSI image version we should leave appVersion the same as the CSI driver version.